fem/restriction.hpp

```diff
@@ -487,34 +531,51 @@ class L2FaceRestriction : public FaceRestriction
    virtual void AddFaceMatricesToElementMatrices(const Vector &fea_data,
                                                  Vector &ea_data) const;
 
+   /** @brief Scatter the degrees of freedom, i.e. goes from L-Vector to
+      face E-Vector.
+
+      @param[in]  x The L-vector degrees of freedom.
+      @param[out] y The face E-Vector degrees of freedom with the given format:
+                    if L2FacesValues::DoubleValued (face_dofs x vdim x 2 x nf)
+                    if L2FacesValues::SingleValued (face_dofs x vdim x nf)
```

Comment:
Currently, this method does not do anything different when `m == L2FacesValues::SingleValued` compared to `m == L2FacesValues::DoubleValued` -- `m` is ignored. Is this comment meant to describe future intended behavior? If that's the case, let's add `MFEM_VERIFY` that `m == L2FacesValues::DoubleValued` with appropriate error message. If the intent (now and in the future) is to not use `m` then let's fix this documentation.

Reply:
I think we always want double-valued. Comment is fixed in 8c85e850c321e78df446949582091626b6fc7007.